sdk/healthinsights/health-insights-radiologyinsights-rest/vitest.config.ts

```diff
@@ -8,8 +8,15 @@ export default mergeConfig(
   viteConfig,
   defineConfig({
     test: {
-      testTimeout: 1200000,
-      hookTimeout: 1200000,
+      typecheck: {
+        enabled: true,
+        tsconfig: "tsconfig.test.json",
+        include: ["test/**/*.ts", "test/**/*.mts", "test/**/*.cts"],
+      },
+      include: ["test/**/*.spec.ts"],
+      hookTimeout: 5000000,
+      testTimeout: 5000000,
+      environment: "node",
```

Comment:
if you just need to override timeouts:
```suggestion
      hookTimeout: 5000000,
      testTimeout: 5000000,
      environment: "node",
```
curious why `environment: "node",` is needed

Reply:
Probably not needed anymore since I skip the browsertest now, but when I did not do this, I added the node environment. Perhaps I should remove this again?